.github/workflows/lighting.yml

```diff
@@ -0,0 +1,17 @@
+name: pytorch_lightning 
+on:
+  push:
+    branches:
+      - '*'
+  schedule:
+    - cron: '*/14 3 * * *' # Run once daily at arbitrary time at night
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
@pitercl Let me know if testing with additional OS's and Python version is desirable.